spinalcordtoolbox/deepseg/models.py

```diff
@@ -191,6 +174,15 @@
          "thr": None,  # Images are already binarized
          "default": False,
      },
+    "model_seg_ms_lesion_mp2rage": {
+         "url": [
+             "https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20240610/nnUNetTrainer_seg_ms_lesion_mp2rage__nnUNetPlans__3d_fullres.zip"
+         ],
+         "description": "Segmentation of spinal cord MS lesions on MP2RAGE UNIT1 contrast",
+         "contrasts": ["UNIT1"],
```

Comment:
I notice that we used to have the `'mp2rage'` contrast for the old ivadomed model? And, actually, we still do use `mp2rage` for the old spinal cord model, too. 

> Note: The exact value for `contrasts` doesn't have any functional purpose (since `-c` is only considered for two models) -- its only use is in the documentation (`-list-tasks`) to guide users.

To keep the models consistent, should we use `mp2rage` or should we use `UNIT1`? If we want to use `UNIT1`, then maybe we should also update the `contrasts` field of the other MP2RAGE model?

Reply:
I think we should use `UNIT1`, because we have datasets (ie. `nih-ms-mp2rage` , `hc-leipzig-7t-mp2rage`) with more than `UNIT1` contrast (ie. `T1map`, `desc-denoising_UNIT1`).